Makefile

```diff
@@ -307,10 +307,9 @@ docserve: gen-docs
 	$(RUN) mkdocs serve
 
 
-# Regenerate from Google Sheets. Note that this uses a *separate* Pipenv in the
-# generators/google-sheets directory, so we have to run pipenv install on it separately.
+# Generate LinkML schema from Google Sheet data model using sheet2linkml.
 generate-model: install install-dev
-	CDM_GOOGLE_SHEET_ID=$(CDM_GOOGLE_SHEET_ID) $(RUN) python vendor/sheet2linkml/sheet2linkml.py --output model/schema/crdch_model.yaml
+	CDM_GOOGLE_SHEET_ID=$(CDM_GOOGLE_SHEET_ID) $(RUN) sheet2linkml --output linkml_model/schema/crdch_model.yaml --logging-config ./logging.ini
```

Comment:
I haven't run this within this repo, but I have run it directly in `cancerDHC/sheet2linkml`, so I trust it

Reply:
I think the output file should be `--output model/schema/crdch_model.yaml`, unless we're moving the directory we store crdch_model.yaml in?